packages/react-instantsearch/src/ui/__tests__/HitsPerPage.test.tsx

```diff
@@ -1,149 +0,0 @@
-/**
- * @jest-environment jsdom
- */
-
-import { render } from '@testing-library/react';
```

Comment:
I think this test has some value, no? In the future if we reuse the ui component for all flavours, this will be public api and thus should be tested. Maybe this file should be kept

Reply:
Technically, we should be the only ones using them. Even if they're exposed from the package, they remain an internal component that shouldn't be used for other purposes, and would already be tested via widget tests. The only thing that could "suffer" is code coverage.

I see the point for facilitating versioning though. Let's maybe then only keep the RIS ones and add a comment on all UI tests to specify we're only keeping them from the perspective of migrating UI components as shared APIs. WDYT?

Reply:
I think that makes sense. I see the argument for not having ui tests as well as they overlap though. Happy either way